draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,50 @@ IDNA2003 {{RFC3490}}.
+
+Note that cookies are not the only mechanism that can be used to track users across sites, so while these mitigations are necessary to improve Web privacy, they are not sufficient on their own.
+
 
 ## Third-Party Cookies {#third-party-cookies}
 
-Particularly worrisome are so-called "third-party" cookies. In rendering an HTML
-document, a user agent often requests resources from other servers (such as
-advertising networks). These third-party servers can use cookies to track the
-user even if the user never visits the server directly. For example, if a user
-visits a site that contains content from a third party and then later visits
-another site that contains content from the same third party, the third party
-can track the user between the two sites.
-
-Given this risk to user privacy, some user agents restrict how third-party
-cookies behave, and those restrictions vary widly. For instance, user agents
-might block third-party cookies entirely by refusing to send Cookie header fields or
-process Set-Cookie header fields during third-party requests. They might take a less
-draconian approach by partitioning cookies based on the first-party context,
-sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another. Or they might even allow some
-third-party cookies but block others depending on user-agent cookie policy or
-user controls.
-
-This document grants user agents wide latitude to experiment with third-party
-cookie policies that balance the privacy and compatibility needs of their users.
-However, this document does not endorse any particular third-party cookie
-policy.
-
-Third-party cookie blocking policies are often ineffective at achieving their
-privacy goals if servers attempt to work around their restrictions to track
-users. In particular, two collaborating servers can often track users without
-using cookies at all by injecting identifying information into dynamic URLs.
-
-## Cookie policy {#cookie-policy}
+In a Web browsing context, a "third-party" or cross-site cookie is a cookie that is associated with
+content that is served from a different server than the one that hosts the primary resource (that
+is, the HTML). HTML documents can embed content (such as scripts, images, stylesheets, frames) that
+is served from other domains. These third-party resources can use cookies to track the user even if
+they never visit that server directly. For example, if a user visits a site that contains content
+from a third party and then later visits another site that contains content from the same third
+party, the third party can track the user between the two sites.
```

Comment:
```suggestion
party, that third party might be able to use cookies to track the user between the two sites.
```

Reply:
what value does 'might be able to' add?

Reply:
...because "can" implies an absolute possibility (I can use legs to climb stairs), in this case whether tracking is possible depends on what policies are in place with respect to cross site cookies.

Reply:
This para is already super-wordy. Will try to refactor.